server/backend/mlx.py

```
@@ -767,5 +408,273 @@ def build_harmony_conversation(
+        "reasoning", id, text, output_index, sequence_number
+    )
+    return (
+        resp_str_a + resp_str_b + resp_str_c,
+        sequence_number,
+        output_index,
+        item_chunk,
+    )
+
+
+def _process_output_item_done(
+    type: str, id: str, final_text: str, output_index, sequence_number: int
```

Ruff (0.15.13)

[error] Function argument `type` is shadowing a Python builtin

(A002)